sql/src/main/java/org/opensearch/sql/sql/parser/AstBuilder.java

```diff
@@ -111,6 +111,7 @@ public UnresolvedPlan visitQuerySpecification(QuerySpecificationContext queryCon
     if (queryContext.limitClause() != null) {
       from = visit(queryContext.limitClause()).attach(from);
     }
+    //TODO needs better implementation to create child unnest node.
```

Comment:
Maybe: revert all changes not related to tests?

Reply:
Yes. I'll have to rebase from `dev-spike-nested`. Thanks